pkg/i2gw/providers/istio/converter.go

```diff
@@ -728,7 +748,7 @@ func (c *converter) createHTTPRoute(params createHTTPRouteParams) *gatewayv1.HTT
 // 2. for exact match and for regex match, istio rewrites full URI path to the given value.
 //
 // Also, in K8S Gateway API only 1 HTTPRouteFilterURLRewrite is allowed per HTTPRouteRule
-// https://github.com/kubernetes-sigs/gateway-api/blob/0ad0daffe8d47f97a293b2a947bb3b2ee658e967/apis/v1/httproute_types.go#L228
+// https://github.com/kubernetes-sigs/	-api/blob/0ad0daffe8d47f97a293b2a947bb3b2ee658e967/apis/v1/httproute_types.go#L228
```

Comment:
is this intentional ?

Reply:
No this must have happened by mistake, i ll revert